src/Sulu/Bundle/CategoryBundle/DependencyInjection/SuluCategoryExtension.php

```diff
@@ -119,6 +119,22 @@ public function prepend(ContainerBuilder $container)
                                 ],
                             ],
                         ],
+                        'single_selection' => [
+                            'single_category_selection' => [
+                                'default_type' => 'list_overlay',
+                                'resource_key' => 'categories',
+                                'types' => [
+                                    'list_overlay' => [
+                                        'adapter' => 'tree_table_slim',
+                                        'list_key' => 'categories',
+                                        'display_properties' => ['name'],
+                                        'empty_text' => 'sulu_category.no_category_selected',
+                                        'icon' => 'fa-tags',
```

Comment:
did you set this on purpose?

Reply:
yes have not found any better icon and we also have not used any other icon for categories in the whole system